### ISSUE TITLE
Log full requests body ?

### ISSUE THREAD

**Issue by bflorat** [outofcoffee/imposter#210]
I read the Metrics/telemetry/Logs documentation and I'm able to log requests details but not requests full bodies (required by our developers  for debugging purposes).

Is there a way to achieve this ?

**Comment by outofcoffee** [comment]
Hi @bflorat thank you for getting in touch.

The ability to optionally include request/response body in the logs is a good enhancement, so we can add that feature.

In the meantime, as a stopgap, you could do the logging using a script, e.g.

```yaml
# your-config.yaml

plugin: rest

# add a default script file to all resource responses
defaultsFromRootResponse: true
response:
  scriptFile: logger.js

# ...other config
```

```js
// logger.js

var req = context.request;
logger.debug('Request body for: ' + req.uri + ': '  + req.body);
```

**Comment by outofcoffee** [comment]
Released in [v3.1.0](https://github.com/outofcoffee/imposter/blob/main/CHANGELOG.md#310---2022-09-08)

Docs here: https://docs.imposter.sh/metrics_logs_telemetry/#logging-requestresponse-body

**Comment by bflorat** [comment]
Great! thanks a lot.  KUTGW
